py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -818,6 +818,7 @@ def handle_antiklepto(request: eth.ETHRequest) -> bytes:
                     recipient=recipient,
                     value=value,
                     data=data,
+                    address=recipient.hex(),
```

Comment:
the method itself should take an `address` parameter so that users of the library can provide the address as the user typed it in. by adding `recipient.hex()` here, you force the display of the address on the bitbox02 to be lowercase, which is not the goal.

Reply:
Then how can I take the `address` as the user types in? I couldn't see a direct way to get that. Only thing I saw was all fields of `request`, including `recipient`, was decoded from bytes of `transaction` that was given as input.

Reply:
the address as the user types it (or the enum value if we go for the enum solution, which i think we should) would be provided by the user of the client library. the user of the client library, e.g. the BitBoxApp or Rabby (or any wallet) knows what the user typed and can infer the right value.